{"uuid": "cdbc307d-2195-4f77-b2de-2ca7d612a4b8", "children": ["de167dd3-b57b-426c-852c-abe5953ed09e"], "befores": [{"name": "base_api_acs", "status": "broken", "statusDetails": {"message": "TypeError: BaseAPIACS.__init__() missing 2 required positional arguments: 'base_url' and 'org_id'\n", "trace": "  File \"C:\\Users\\chan\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\chan\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"C:\\Users\\chan\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\chan\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\chan\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\actions-runner\\actions-runner-win-x64-2.324.0\\_work\\api_automation\\api_automation\\conftest.py\", line 65, in base_api_acs\n    return (BaseAPIACS(env=env), endpoint)\n"}, "start": 1749801516605, "stop": 1749801516605}], "start": 1749801516605, "stop": 1749801516812}